src/tabs/borderless.scss

```diff
@@ -4,49 +4,55 @@
 @import './tabs';
 
 @mixin iui-tabs-borderless {
-  @include iui-tabs;
-  margin: $iui-baseline/2 0;
+  .iui-tab {
+    height: $iui-component-height;
+    padding: 0 ($iui-component-padding-horizontal * 2);
+    background-color: transparent;
 
-  > li {
-    > a {
-      line-height: $iui-line-height;
-      padding: $iui-line-height $iui-l;
-
-      &:hover {
-        @include themed {
-          background-color: rgba(
-            t(iui-color-foreground-primary-rgb),
-            t(iui-opacity-6)
-          );
-        }
-      }
-
-      &::after {
-        bottom: $iui-xxs;
-        left: $iui-xxs;
-        right: $iui-xxs;
-      }
-    }
-
-    &.iui-tabs-active > a {
+    &:hover {
       @include themed {
         background-color: rgba(
           t(iui-color-foreground-primary-rgb),
           t(iui-opacity-6)
         );
       }
     }
-  }
 
-  &.iui-green > li {
-    > a:hover,
-    &.iui-tabs-active > a {
+    &.iui-active {
```

Comment:
I think that would be really cool on pill and borderless tabs, similar to [this](https://material-components.github.io/material-components-web-catalog/#/component/tabs).

Reply:
If we want to have animation with a stripe then we need to bring back `::after` 😀

Reply:
This is the example of what we could do:
![tab-animation](https://user-images.githubusercontent.com/36186912/122352198-90994400-cf57-11eb-9a36-fc8e7fca1344.gif)
With this code:
![image](https://user-images.githubusercontent.com/36186912/122352180-8b3bf980-cf57-11eb-9ecf-a8f33918a3cb.png)

Reply:
If we use a separate div for the active stripe, we could have an animation like material (from Jon's link).

Reply:
I think the animation from material design would look good on borderless & pill tabs, but would not look as nice on the default tabs because of the clear dividing lines.  I'd be ok with using Materials animation on borderless & pills but using Vilius animation on default.  Would that be too sloppy?

Reply:
they're different styles so different animations make sense